universum/modules/launcher.py

```diff
@@ -282,8 +282,9 @@ def finalize(self) -> None:
                 text = utils.trim_and_convert_to_unicode(text)
                 if self.file:
                     self.file.write(text + "\n")
-                self.add_tag(self.configuration.fail_tag)
-                self._error = text
+                if not self.configuration.is_conditional:
```

Comment:
Is the logic that since the conditional step always succeeds (kind of), there is no sense to use tagging for it?

Reply:
Ah, nope. The "pass_tag" is supported though.
Looks like inconsistency.